bookmarklet.js

```diff
@@ -53,6 +69,7 @@ void(
     });
 
     // fetch assignments and parse into Event objects
+    // (pr) curious -- what's the "ignoreThis" param doing here (and on assignment_groups)?
```

Comment:
That's a pretty hacky way of making the urls okay for the `paginated_fetch` function, since that function adds pagination to the url with an & and will fail on a request with no GET params. Since the Canvas API igonres things it doesn't understand, I just added a nonsense filler param.